conjur/controller/init_controller.py

```diff
@@ -66,25 +74,40 @@ def get_server_certificate(self):
                 # pylint: disable=raise-missing-from
                 raise RuntimeError("Error: URL is required")
 
+    # TODO: Factor out the following URL validation to ConjurrcData class
+    def format_conjur_url(self):
+        """
+        Method for formatting the Conjur server URL to
+        break down the URL into segments
+        """
         # Chops off the '/ if supplied by the user to avoid a server error
         if self.conjurrc_data.conjur_url.endswith('/'):
-            self.conjurrc_data.conjur_url=self.conjurrc_data.conjur_url[:-1]
+            self.conjurrc_data.conjur_url = self.conjurrc_data.conjur_url[:-1]
 
-        url = urlparse(self.conjurrc_data.conjur_url)
+        return urlparse(self.conjurrc_data.conjur_url)
 
-        # TODO: Factor out the following URL validation to ConjurrcData class
-        # and add integration tests
-        if url.scheme != 'https':
+    def validate_conjur_url(self, conjur_url):
+        """
+        Method for validating the Conjur server URL to
+        ensure it is provided as expected
+        """
+        if conjur_url.scheme != 'https':
             raise RuntimeError(f"Error: undefined behavior. Reason: The Conjur URL format provided "
                    f"'{self.conjurrc_data.conjur_url}' is not supported.")
 
+    # pylint: disable=line-too-long
+    def get_server_certificate(self, conjur_url):
+        """
+        Method to get the certificate from the Conjur endpoint detailed by the user
```

Comment:
IMHO we can drop the `Method to` prefix

```
Gets the certificate from the specified conjur_url
Return tuple of certificate fingerprint and certificates chain or None in case conjurrc contains a certificate file
```